test/codegen/newman/fixtures/formdataFileCollection.json

```diff
@@ -52,7 +55,7 @@
               "key": "binary file",
               "value": "",
               "type": "file",
-              "src": "<file path>"
+              "src": "/mnt/c/Users/INVICTUS/Desktop/GSoC/postman-code-generators/dummyBinaryFile"
```

Comment:
this should be `<file path>` There is a script which takes care of adding the path from the CI here.